guide/payments/hide-sensitive-info.md

```diff
@@ -0,0 +1,107 @@
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Hiding Sensitive Information
+
+Imagine this scenario. You are in a public place, and you need to make a payment using your bitcoin wallet. You open your wallet on your phone, but you don’t feel comfortable having your address and balance information clearly visible to strangers who may be looking over your shoulder. Hence by giving users the ability to hide sensitive information in their wallet if desired, they gain an added sense of privacy and security when using the app in public.
+
+## What information is considered sensitive ?
+
+Sensitive information in wallet applications include the wallet balance, addresses, private keys and previous transactions information. They are considered sensitive for the following reasons, the wallet balance is considered sensitive because it shows exactly how much you have in your wallet, addresses and previous transaction information are sensitive because they can be used to track how you spend and receive your bitcoins, and private keys, if they fall into the wrong hands, can be used to access and transfer your bitcoins.
+
+It's more common for wallets to protect private keys but not much is done for other sensitive information like the balance, addresses and previous transactions but a few wallets like Bitcoin Core, Wasabi, Wallet of Satoshi, and others have made it work though. Below describes a pattern, and considerations for hiding and revealing sensitive information
+
+## Hiding and revealing information with the hide icon
+
+The hide icon / button, which is usually displayed within close reach of the balance itself, is used to quickly and easily hide and reveal wallet information by tapping or clicking on it.
+
+![hidden by eye icon](/assets/images/payments/hide-info-hidden-by-eye-icon.png)
+
+Pros
+
+- It is quite convenient to switch between revealed and hidden state
+
+Cons
+
+- Super easy for anyone else to reveal your information if they have access to your device.
+
+
+## Entering a pin to unveil information
+
+A problem that seemed to arise throughout each implementation was that is was as easy to reverse the hidden state as it was to enable it. This may be for convenience sake, however, if you are in a situation where you are concerned about unauthorized access by someone who has your device, once hidden perhaps you should be only able to unhide if a PIN or password has been entered. This could therefore reaffirm the identity of the wallet owner for extra security.
+
+![reveal by pin](/assets/images/payments/hide-info-reveal-by-pin.png)
+
+Pros
+- The risk of an unauthorized person revealing your information is minimal due to the pin required
+
+Cons
+
+- It may be annoying having to repeatedly put in your min when ever you want to reveal your information especially if you do so often
```

Comment:
>  - It may be annoying having to repeatedly put in your min when ever you want to reveal your information especially if you do so often 

Too informal imo. `Annoying`, also unsure what this means ` put in your min when ever you want to reveal your information `